python/recipe.cpp

```diff
@@ -110,9 +110,31 @@ std::vector<arb::event_generator> py_recipe_shim::event_generators(arb::cell_gid
     return gens;
 }
 
+struct cell_connection_shim {
+    arb::cell_member_type source;
+    arb::cell_member_type dest;
+    float weight;
+    float delay;
```

Comment:
For consistency we should use `arb::time_type` for `delay`. I see that in the main Arbor library we use `float`, instead of `time_type`. First fix that, then use `time_type` here.